app/src/main/java/co/touchlab/droidconandroid/shared/tasks/persisted/RefreshScheduleJob.java

```diff
@@ -42,16 +42,21 @@ public void onAdded()
     @Override
     public void onRun() throws Throwable
     {
+        // Could extract all of this stuff out for testing's sake
```

Comment:
Since this job needs to persist, we cant pass in anything thats not serializable